src/Translator.php

```diff
@@ -455,25 +454,31 @@ private function removeEnglishStringsFromJsonTranslations(): void
     {
         $this->log('Removing english from json translation files');
         $count = 0;
-        foreach (array_keys($this->originalJson) as $path) {
-            if (!file_exists($path)) {
-                continue;
-            }
-            $enPath = preg_replace('#/[^/\.]+\.json$#', '/en.json', $path);
-            $enJson = $this->jsonDecode(file_get_contents($enPath));
-            if ($enPath === $path) {
-                continue;
-            }
-            // Remove any keys where the value is the same as the source english value
-            $contentJson = $this->jsonDecode(file_get_contents($path));
-            foreach (array_keys($contentJson) as $key) {
-                if (array_key_exists($key, $enJson) && $enJson[$key] === $contentJson[$key]) {
-                    unset($contentJson[$key]);
-                    $count++;
+        foreach ($this->modulePaths as $modulePath) {
+            $jsPaths = $this->getJSLangDirectories($modulePath);
+            foreach ((array)$jsPaths as $jsPath) {
+                foreach (glob("{$jsPath}/src/*.js*") as $path) {
+                    $ext = pathinfo($path, PATHINFO_EXTENSION);
+                    $enPath = preg_replace('#/[^/\.]+\.js[a-zA-Z]*$#', '/en.' . $ext, $path);
+                    if (!file_exists($enPath)) {
+                        $enPath = preg_replace('#/[^/\.]+\.js[a-zA-Z]*$#', '/en.json', $path);
+                    }
```

Comment:
This is also new - userforms has their src files as `*.js` files, not `*.json` files. So check for the en file with the same ext first, then try json explicitly as a fallback,